### PR TITLE
Page for xml datatype

### PR DESCRIPTION
**Pull request by janebeckman**
New page includes xml datatype and xml function material.

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/query/topics/xml-data.xml [hunk 1]
```diff
@@ -0,0 +1,625 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="topic1" xml:lang="en">
+  <title id="ih135496">Working with XML Data</title>
+  <body>
+    <p>Greenplum Database supports the <codeph>xml</codeph> data type that stores XML data. The
+        <codeph>xml</codeph> data type checks the input values for well-formedness, providing an
+      advanate over stimply storing XML data in a text field. Additionally, support functions allow
```
Comment (hlinnaka) [inline gpdb-doc/dita/admin_guide/query/topics/xml-data.xml]: advante -> advantage

Comment (hlinnaka) [inline gpdb-doc/dita/admin_guide/query/topics/xml-data.xml]: stimply -> simply

File: gpdb-doc/dita/admin_guide/query/topics/xml-data.xml [hunk 2]
```diff
@@ -0,0 +1,625 @@
...
+        If no particular namespace is wanted, an empty string should be passed.</p>
+      <p>The following functions return XML Schema documents describing the mappings performed by
+        the corresponding functions
+        above:<codeblock>able_to_xmlschema(tbl regclass, nulls boolean, tableforest boolean, targetns text)
+query_to_xmlschema(query <varname>text</varname>, nulls boolean, tableforest boolean, targetns text)
+cursor_to_xmlschema(cursor refcursor, nulls boolean, tableforest boolean, targetns text)
+</codeblock></p>
+      <p>It is essential that the same parameters are passed in order to obtain matching XML data
+        mappings and XML Schema documents.</p>
+      <p>The following functions produce XML data mappings and the corresponding XML Schema in one
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/query/topics/xml-data.xml]: lower-case "schema"

File: gpdb-doc/dita/admin_guide/query/topics/xml-data.xml [hunk 3]
```diff
@@ -0,0 +1,625 @@
...
+    <title>XML Functions</title>
+    <body>
+      <p>The functions and function-like expressions described in this section operate on values of
+        type xml. This section also contains the following sub-topics:<ul id="ul_ypx_f4w_2z">
+          <li><xref href="#topic_zpg_jl2_wz" format="dita"/></li>
+          <li><xref href="#topic_xvc_1vd_wz" format="dita"/></li>
+          <li><xref href="#topic_z5y_nxd_wz" format="dita"/></li>
+        </ul></p>
+      <p>The following functions are supported.</p>
+      <p><cmdname>xmlcomment</cmdname></p>
```
Comment (janebeckman) [inline gpdb-doc/dita/admin_guide/query/topics/xml-data.xml]: I was trying to figure how to best organize these functions, myself. I think the idea of moving it to the end and putting it in a dedicated reference section, as you suggest, seems like a reasonable way of handling it.

**Comment by dyozie** [comment]
Thanks, Jane - I'm going to merge this and then see about possibly reformatting some of the reference info.

**Pull request merged**